lib/interfaces/heroku_interface.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../http_post_interface'
+Dir['./lib/slack_commands/*.rb'].each { |file| require file }
+
+class HerokuInterface < HttpPostInterface
+  def handle_heroku_webhook(payload)
+    params = Rack::Utils.parse_nested_query(payload)
+    user = params['user'].split('@')[0]
+    release = "#{params['release']} " if params['release']
+    server = params['url'].split('/').last.split('.').first[8..-1]
```

Comment:
If you're just trying to get the domain, it'd probably be more robust to use a dedicated url parser:
```
uri = URI.parse('https://github.com/kkuchta/bub/pull/3/files')
uri.host  # "github.com"
```

And, although this is more debatable, I'd argue that a regex would be a bit more readable than `.split('.').first[8..-1]`, eg `"joyable-sassy.heroku".match('joyable-(.+)')[1]`.